search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -243,18 +255,80 @@
     time
     gran-time))
 
+(defn- title->citation-title
+  "Format citation name."
+  [title version]
+  (when title
+    (str title
+         (when version
+           (str ". Version " version)))))
+
+(defn- publisher->citation-publisher
+  "Format publisher for citation."
+  [archive-center provider-id]
+  (when archive-center
+    (str "Archived by "
+         (if (= provider-id "USGS_EROS")
```

Comment:
We should have a variable defined rather than using "USGS_EROS". I think there's another place in this file that checks for the same string.
We should define this as a variable (and I think there's somewhere else in the code that checks for USGS as well).
```
(def USGS_PROVIDER "The USGS provider ID." "USGS_EROS")
```

Is LPDAAC_ECS also a USGS provider rather than NASA? Are there any other providers that we shouldn't be treating as NASA? We might need to talk to Scott, Erich or Chris Lynnes.